sirepo/pkcli/admin.py

```diff
@@ -141,9 +135,87 @@ def move_user_sims(target_uid=""):
         shutil.move(lib_file, target)
 
 
+def _build_ops(ops, simulations, sim_type, examples):
+    n = set([x.models.simulation.name for x in examples[sim_type]])
+    for sim in simulations:
+        if sim.name not in n:
+            ops.delete.append((sim, sim_type))
```

Comment:
Does this delete all simulations, not just all simulation examples?

Reply:
`simulations` name made it seem like all simulations when it was already filtered by examples